index.rst

```diff
@@ -1,42 +1,621 @@
-..
-  Content of technical report.
+:tocdepth: 1
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Notation and Conventions
+========================
 
-  Use the following syntax for sections:
+Given multiple overlapping images :math:`\{z_1({\bf r}), z_2({\bf r}), ...\}`, we designate by :math:`R({\bf r})` the set of image indices that overlap a point on the sky :math:`{\bf r}`.  We assume that all images have already been resampled to a common coordinate system and common photometric units.  Each image is related to the true sky :math:`f({\bf r})` via its point spread function (PSF) :math:`\phi_i({\bf r}, {\bf s})` by
```

Comment:
I though about this, but I don't like how much it clutters up the notation.  My hope was that this section would essentially explain in prose that it's discrete so I don't have to do that.